### PR TITLE
Bug 2012780: Avoid dynamically allocated port range for haproxy

### PR DESCRIPTION
**Pull request by cybertron**
The haproxy stats and health endpoints were configured with high
port numbers (50000 and 50936) that conflict with the range used for
dynamic allocation of ports:

$ cat /proc/sys/net/ipv4/ip_local_port_range
32768   60999

This can occasionally cause haproxy to fail if the kernel happens to
pick one of those ports for a client connection.

To avoid this, we should use ports outside the range shown above.
This patch changes the health endpoint to 30936, while the stats
port is controlled by runtimecfg and will be fixed in a separate
commit there.

**- What I did**

**- How to verify it**

**- Description for the changelog**
Moved the haproxy health check port to avoid conflicts with dynamically allocated client ports.

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
Looks good except for commit message that mentions port 50000.

**Review by sinnykumari — APPROVED** [review]
LGTM.
/hold
Feel free to remove hold once all the relevant team has LGTM'ed

**Comment by cybertron** [comment]
/retest-required
/hold cancel

CI is passing on all platforms and we have lgtm from 3/4. Should be good to go.

**Pull request merged**
